roles/cluster_infra/templates/resources.tf.j2

```diff
@@ -131,7 +131,7 @@ resource "openstack_compute_instance_v2" "cluster_server" {
     ssh_authorized_keys:
       - {{ cluster_deploy_ssh_public_key }}
 {% if cluster_user_ssh_public_key is defined and cluster_user_ssh_public_key %}
-  users:
+  azimuth_users:
```

Comment:
Hmm, this hard depends on specific image changes, eek... Needs a careful rollout. Probably time to bump the major version when we release this one.

Reply:
I don't think this is an issue - azimuth-ops releases combine a particular playbook version with a particular images version. Just so long as we can upgrade from one to the other.